Gemfile

```diff
@@ -5,4 +5,7 @@ gem "oj", "3.5.1"
 gem "fluent-plugin-elasticsearch"
 gem "fluent-plugin-kubernetes_metadata_filter"
 gem "ffi"
-gem "fluent-plugin-systemd"
\ No newline at end of file
+gem "fluent-plugin-systemd"
+gem "fluent-plugin-concat", "2.1.0"
```

Comment:
why do we need this plugin? I remember it creates unnecessary issues

Reply:
We haven't removed the use of this plugin yet. Once it is removed from the config, I'll remove it from here as well

Reply:
but in this repo there isn't any config which uses this plugin or I am missing something?